src/transformers/convert_slow_tokenizer.py

```diff
@@ -832,13 +833,32 @@ def converted(self) -> Tokenizer:
                 continuing_subword_prefix="",
                 end_of_word_suffix="</w>",
                 fuse_unk=False,
+                unk_token=str(unk_token),
             )
         )
 
-        tokenizer.pre_tokenizer = pre_tokenizers.ByteLevel(add_prefix_space=self.original_tokenizer.add_prefix_space)
+        tokenizer.normalizer = normalizers.Sequence(
+            [normalizers.NFC(), normalizers.Replace(Regex(r"\s+"), " "), normalizers.Lowercase()]
+        )
+        tokenizer.pre_tokenizer = pre_tokenizers.Sequence(
+            [
+                pre_tokenizers.Split(
+                    Regex(r"""'s|'t|'re|'ve|'m|'ll|'d|[\p{L}]+|[\p{N}]|[^\s\p{L}\p{N}]+"""),
+                    behavior="removed",
+                    invert=True,
+                ),
+                pre_tokenizers.ByteLevel(add_prefix_space=False),
+            ]
+        )
```

Comment:
Comment 1

Reply:
For the pre-tokenization step, I reused the regex used by the original tokenizer to pre-tokenize the text before applying BPE (corresponding [code](https://github.com/huggingface/transformers/blob/v4.15.0/src/transformers/models/clip/tokenization_clip.py#L318-322) in the slow version). I just removed the special tokens from this regex as they will already be isolated by the fact that they are added tokens.

As explained in my point 4. above, I also forced `add_prefix_space` to `False`.